backends/candle/tests/test_modernbert.rs

```diff
@@ -202,3 +202,61 @@ fn test_modernbert_classification() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
```

Comment:
Having a bit of trouble running this test and not super pleased with this test but it;s just to test that both cls/mean produce different outputs as expected.

Reply:
How about just testing with the [reranker-ModernBERT-large-gooaq-bce](https://huggingface.co/tomaarsen/reranker-ModernBERT-large-gooaq-bce) model directly, which is a mean classifier-pooling, instead of `gte-reranker` model with mean pooling?

I think we can check whether classifier pooling correctly works by running the `reranker-ModernBERT-gooqa-bce` model, while `cls` classifier pooling has already been verified in the above test! (implicitly assume cls and mean will behave differently)

> unable to download the HF model, hitting a 429, 

when I faced that issue, in my case, I manually download the model via git or huggingface-cli on my local, and then set `HUGGINGFACE_HUB_CACHE` to point to that local path.

Reply:
It couldn't find from the cache but I managed to just export a hugginface token, all good!

```shell
successes:
    test_modernbert_classification_mean_pooling

test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 3 filtered out; finished in 3.94s
```